### ISSUE TITLE
operationalize for basic k8s deployment

### ISSUE THREAD

**Issue by cazlo** [cazlo/Auto_Jobs_Applier_AIHawk#5]
define k8s resources like:
- service + deployment for the react app (probably just nginx hosting a built/minified version of the site)
- service + deployment for the backend API
- deployment for the queue worker
- service + deployment for the queue and DB

depends on #4

**Comment by cazlo** [comment]
also log to stdout instead of a file.

a ton of the logging seems to be to a file instead of stdout, and I would prefer to do this in a standard "12 factor app" way and not have to deal with mgmt of local state (e.g. rotating logs)

**Comment by cazlo** [comment]
closed won't do. see https://github.com/cazlo/Auto_Jobs_Applier_AIHawk/issues/1 for reasoning

all of the above would still be applicable to the new project, but github seemingly doesn't allow moving issues between projects so I will open a copy of this issue in new project
